features/step_definitions/minions_steps.rb

```diff
@@ -0,0 +1,57 @@
+def click_n_refresh(link)
+  fail if not page.has_xpath?("//a[@href='#{link}']")
+  find('#pending-list').find(:xpath, "//a[@href='#{link}']").click
+  sleep(5)
+  visit "/rhn/YourRhn.do"
+  visit "/rhn/manager/minions"
+end
+
+Then(/^I should see a this client as a minion in the Pending section$/) do
+  fail if not find('#pending-list').find("b", :text => @this_client_hostname).visible?
```

Comment:
Whilst we had this discussion, I am not convinced how this makes any significant changes to a boolean _visible_ or _invisible_. But I indeed would reconsider this for `assert`, after Cynthia hammers all the UI and this whole test will be changed. Let this stay for now like that.